tpm2/tpm2_test.go

```diff
@@ -325,6 +361,11 @@ func TestLoadExternalPublicKey(t *testing.T) {
 		run(t, rp, private)
 	})
 	t.Run("ECC", func(t *testing.T) {
+		if ok, err := supportsECC(rw); !ok && err == nil {
```

Comment:
Change `supportsECC` to `skipOnUnsupportedAlg(t, rw, AlgECC)`, to remove duplicated checks.